### PR TITLE
Chef Infra Client 15 Release Notes Additional edits

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -224,13 +224,13 @@ In order to accommodate a combined bootstrap that supports both SSH and WinRM, s
...
+| --session-timeout MINUTES | --session-timeout SECONDS|New for ssh, existing for winrm. The unit has changed from MINUTES to SECONDS for consistency with other timeouts. |
 | --ssh-password | --connection-password | |
 | --ssh-port | --connection-port | `knife[:ssh_port]` config setting remains available.
 | --ssh-user | --connection-user | `knife[:ssh_user]` config setting remains available.
 | --ssl-peer-fingerprint | --winrm-ssl-peer-fingerprint | |
-| --prerelease |--channel CHANNEL | This now allows you to specify the channel that Chef Infra Client gets installed from. Valid values are _stable_, current, and unstable. 'current' has the same effect as using the old --prerelease. |
-| --winrm-authentication-protocol=PROTO | --winrm-auth-method=AUTH-METHOD | Valid values: plaintext, kerberos, ssl, _negotiate_|
+| --prerelease |--channel CHANNEL | This now allows you to specify the channel that Chef Infra Client gets installed from. Valid values are _stable_, _current_, and _unstable_. 'current' has the same effect as using the old --prerelease. |
```
Comment (mjingle) [inline RELEASE_NOTES.md]: @tas50 Originally, just "stable" was italicized. I added the same formatting to the other valid values. Not sure of the reason for the italics.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -224,13 +224,13 @@ In order to accommodate a combined bootstrap that supports both SSH and WinRM, s
...
+| --winrm-authentication-protocol=PROTO | --winrm-auth-method=AUTH-METHOD | Valid values: _plaintext_, _kerberos_, _ssl_, _negotiate_|
```
Comment (mjingle) [inline RELEASE_NOTES.md]: @tas50 These are also in italics to match "negotiate."  Not quite sure why that was so in the first place.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ See our [Frequently Asked Questions document](https://www.chef.io/bmc-faq/) for
 
 ### Target Mode Prototype
 
-Chef Infra Client 15 adds a prototype for a new method of executing resources called Target Mode. Target Mode allows a Chef Infra Client run to manage a remote system over ssh or another protocol supported by the Train library. This includes platforms that we currently support like Ubuntu Linux, but also allows for configuring other architectures and platforms, such as switches that do not have native builds of Chef Infra Client. Target Mode maintains a separate node object for each target and allows you to manage that node using existing patterns that you currently use.
+Chef Infra Client 15 adds a prototype for a new method of executing resources called Target Mode. Target Mode allows a Chef Infra Client run to manage a remote system over ssh or another protocol supported by the Train library. This support includes platforms that we currently support like Ubuntu Linux, but also allows for configuring other architectures and platforms, such as switches that do not have native builds of Chef Infra Client. Target Mode maintains a separate node object for each target and allows you to manage that node using existing patterns that you currently use.
```
Comment (mjingle) [inline RELEASE_NOTES.md]: Is Target Mode supposed to be `target mode` at any point of this paragraph?

Comment (tas50) [reply RELEASE_NOTES.md]: It's a feature we usually Make Those Marketing Ready

Comment (mjingle) [inline RELEASE_NOTES.md]: Also, would `ssh` supposed to be formatted a certain way when referenced in the second sentence? I see it also typed as "SSH" elsewhere in the Chef Infra Client 15 release notes.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (tas50) [reply RELEASE_NOTES.md]: 👍

File: RELEASE_NOTES.md (hunk 3 above)
Comment (tas50) [reply RELEASE_NOTES.md]: the industry seems to do this however they feel like. We should pick a way. I'm fine with either.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -126,7 +126,7 @@ See the [snap_package](https://docs.chef.io/resource_snap_package.html) document
 
 ### windows_task
 
-windows_task now supports the Start When Available option with a new `start_when_available` property.
+The `windows_task` resource now supports the Start When Available option with a new `start_when_available` property.
```
Comment (mjingle) [inline RELEASE_NOTES.md]: windows_task is a resource, correct? 😬

Comment (tas50) [reply RELEASE_NOTES.md]: yes

Comment (mjingle) [inline RELEASE_NOTES.md]: Also, is the "Start When Available" option supposed to be all one word per https://docs.microsoft.com/en-us/windows/desktop/TaskSchd/taskschedulerschema-startwhenavailable-settingstype-element ?

File: RELEASE_NOTES.md (hunk 3 above)
Comment (mjingle) [inline RELEASE_NOTES.md]: The 3rd sentence, "This support includes...," doesn't really make sense. Originally, "This support" was just "This." I wasn't able to discern what "This" was supposed to be. Are there edits that could be made to clarify this sentence further?

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -168,7 +168,7 @@ Chef now ships with Ruby 2.6.3. This new version of Ruby improves performance an
 
 Platform and platform_family detection on Linux has been rewritten to utilize the latest config files on modern Linux distributions before falling back to slower and fragile legacy detection methods. Ohai will now begin by parsing the contents of `/etc/os-release` for OS information if available. This feature improves the reliability of detection on modern distros and allows detection of new distros as they are released.
```
Comment (mjingle) [inline RELEASE_NOTES.md]: For formatting purposes, does "Platform" or "platform_family" need some `type` of formatting in this paragraph? I wasn't sure if we wanted to include certain annotation or not.

Comment (mjingle) [inline RELEASE_NOTES.md]: Did 'Ohai' get a name change with all the other product name changes?

File: RELEASE_NOTES.md (hunk 3 above)
Comment (mjingle) [reply RELEASE_NOTES.md]: I'm inclined to go with SSH based on what I'm seeing elsewhere.

Comment (mjingle) [inline RELEASE_NOTES.md]:
```suggestion
Chef Infra Client 15 adds a prototype for a new method of executing resources called Target Mode. Target Mode allows a Chef Infra Client run to manage a remote system over SSH or another protocol supported by the Train library. This support includes platforms that we currently support like Ubuntu Linux, but also allows for configuring other architectures and platforms, such as switches that do not have native builds of Chef Infra Client. Target Mode maintains a separate node object for each target and allows you to manage that node using existing patterns that you currently use.
```

Comment (mjingle) [reply RELEASE_NOTES.md]:
```suggestion
Chef Infra Client 15 adds a prototype for a new method of executing resources called Target Mode. Target Mode allows a Chef Infra Client run to manage a remote system over SSH or another protocol supported by the Train library. This support includes platforms that we currently support like Ubuntu Linux, but also allows for configuring other architectures and platforms, such as switches that do not have native builds of Chef Infra Client. Target Mode maintains a separate node object for each target and allows you to manage that node using existing patterns that you currently use.
```

**Comment by mjingle** [comment]
@tas50 in line 298, does "nil" need 'nil' formatting added?

**Comment by mjingle** [comment]
@tas50 On the last sentence of line 384, would the term "Chef Infra server" or "Chef Infra server" be used? It was a topic of debate.

File: RELEASE_NOTES.md (hunk 4 above)
Comment (tas50) [reply RELEASE_NOTES.md]: We change the casing. Microsoft loves camelcase, but that's really just a windows things.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (tas50) [reply RELEASE_NOTES.md]: 👍

File: RELEASE_NOTES.md (hunk 4 above)
Comment (mjingle) [reply RELEASE_NOTES.md]: Understood!

File: RELEASE_NOTES.md (hunk 5 above)
Comment (tas50) [reply RELEASE_NOTES.md]: Ohai did not a rename

Comment (tas50) [reply RELEASE_NOTES.md]: These should probably get formatting

File: RELEASE_NOTES.md (hunk 3 above)
Comment (tas50) [reply RELEASE_NOTES.md]: This support includes -> Target mode supports

File: RELEASE_NOTES.md (hunk 5 above)
Comment (mjingle) [reply RELEASE_NOTES.md]: All, some, or certain instances?

Comment (tas50) [reply RELEASE_NOTES.md]: Those are node attributes not generic references to a platform. We're really talking about node['platform'] and node['platform_family'] so they should get the special treatment.

**Comment by tas50** [comment]
We should call it chef infra server where we can.

**Comment by tas50** [comment]
it would also probably be helpful to stylize the nil. That would help with readability for some.

**Comment by mjingle** [comment]
@tas50 Re: On the last sentence of line 384, the debate was about if the sentence was referring to the literal server that Chef Infra Server would be on, or if it referred to Chef Infra Server the software. I totally forgot to capitalize the S in the second mention!

**Comment by tas50** [comment]
Rebased, merged and the other suggestions are implemented. This is on master now.

**Pull request merged**